datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/assertion/EntityAssertionsResolver.java

```diff
@@ -25,11 +26,16 @@
 import java.util.Objects;
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /** GraphQL Resolver used for fetching the list of Assertions associated with an Entity. */
 public class EntityAssertionsResolver
     implements DataFetcher<CompletableFuture<EntityAssertionsResult>> {
 
+  private static final Logger logger =
```

Comment:
tiny nit:

usually we use lombok

@Slf4j

and then you will be able to do 

log.info()

etc